packages/react-components/react-portal-compat/tsconfig.json

```diff
@@ -1,8 +1,7 @@
 {
   "extends": "../../../tsconfig.base.json",
   "compilerOptions": {
-    "target": "ES5",
-    "lib": ["es5", "dom"],
+    "target": "ES2019",
```

Comment:
Isn't this package used in v8 and v0? If so, I don't think those packages support this target.

Reply:
Compat is intended to be used with v9 components i.e. v8 <=> v9 and v0 <=> v9. This means that you will have v9 components in your bundle and you're fine with this compilation target. If one more package will follow it, this should not be a problem.